CODE_OF_CONDUCT.md

```diff
@@ -40,6 +40,16 @@ Project maintainers are responsible for clarifying the standards of
 acceptable behavior and are expected to take appropriate and fair
 corrective action in response to any instances of unacceptable behavior.
 
+Members of the official team are required to disclose any interests 
+that may conflict against the interests of the privacytools.io website and/or community. 
+If a team member has a private interest (for example, a job at a company which is listed 
+or wants to be listed on the privacytools website), then it is seen as their duty to disclose this interest. 
+
+To determine whether something is a conflict of interest or not, the following question will be asked at a case by case basis: Does the team members interest create an incentive to act (consciously  or sub-consciously) against the best interests of the Privacytools.io website and/or community.
+In short, when a team member has a conflict of interest, they may no longer vote about the subject they are conflicted about
+
+The team member is, however, allowed to join a discussion about the conflicted subject by voicing his own opinion and thoughts, but only if  they declare their conflict of interest with every message, post, call, etc.
```

Comment:
```suggestion
The team member is, however, allowed to join a discussion about the conflicted subject by voicing their own opinion and thoughts, but only if  they declare their conflict of interest with every message, post, call, etc.
```

Reply:
I take it that this means the "request changes" button stays open. However GitHub reactions get closed out?